watertap/costing/units/ion_exchange.py

```diff
@@ -328,24 +340,30 @@ def cost_ion_exchange(blk):
         expr=blk.capital_cost
         == pyo.units.convert(
             (
-                (blk.capital_cost_vessel + blk.capital_cost_resin)
-                * (blk.unit_model.number_columns + blk.unit_model.number_columns_redund)
+                ((blk.capital_cost_vessel + blk.capital_cost_resin) * tot_num_col)
                 + blk.capital_cost_backwash_tank
                 + blk.capital_cost_regen_tank
             )
-            * TIC,
+            * ion_exchange_params.total_installed_cost_factor,
             to_units=blk.costing_package.base_currency,
         )
     )
     if blk.unit_model.config.hazardous_waste:
+        blk.regen_dens = 1000 * pyo.units.kg / pyo.units.m**3
```

Comment:
Does this need to be accessible as a parameter?

Reply:
Its not a bad idea. I imagine you could have some pretty dense regen solutions, but unclear how much it would impact this calculation (hazardous waste disposal), which in my experience is a pretty small part of the costs when included.